networking/portfwd.go

```diff
@@ -59,7 +69,17 @@ func (e *podEnv) forwardPorts(fps []ForwardedPort, defIP net.IP) error {
 	}
 
 	for _, p := range fps {
-		if err = forwardPort(ipt, chain, &p, defIP); err != nil {
+		dst := fmt.Sprintf("%v:%v", defIP, p.PodPort)
+		dstIP := fmt.Sprintf("%v", defIP)
+		dport := strconv.Itoa(int(p.HostPort))
+
+		// Rewrite the destination
+		if err := ipt.AppendUnique("nat", chainDNAT, "-p", p.Protocol, "--dport", dport, "-j", "DNAT", "--to-destination", dst); err != nil {
+			return err
```

Comment:
Seems so, but let's fix this on the outer scope and just call `unforwardPorts()` if `forwardPorts()` fails